articles/monitoring/guides/monitor-applications.md

```diff
@@ -13,20 +13,48 @@ useCase:
   - synthetic-transactions
 ---
 # Monitor Applications
+Auth0 is a critical dependency of your application and/or service. Monitoring Auth0's health can be important as it allows you to report specific errors to your customers and/or take mitigating actions in case of an issue with Auth0.
 
-If you would like to monitor your own [application](/applications) or conduct end-to-end testing, you’ll need to set up your own tests. 
+You can monitor Auth0 using many approaches, each of them complementary with the others. Picking the approaches to use depends on your needs and investment possibilities.
 
-If you've extended Auth0 through [rules](/rules) or a [custom database connection](/connections/database/custom-db), you can build a synthetic transaction that exercises these capabilities using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant). One way of doing this is to [Monitor Auth0 Using SCOM](/monitoring/guides/monitor-using-SCOM).
+## Synthetic transactions
+The simplest approach to monitor Auth0. This involves setting up a periodic request to perform an authentication transaction. If the request works successfully, Auth0 is working fine. If the request fails this _might_ indicate an issue with Auth0.
 
-Auth0 recommends using an authentication flow that doesn't require a user interface such as the **Resource Owner Password Grant**. That way, you can use a monitoring tool that doesn't have to mimick the actions of a user. Many monitoring services exist with this capability including:
+> There _might_ be an issue with Auth0, but the issue could also either be specific to the tenant used for the synthetic transaction or just a single failed request.
 
-* [New Relic](http://newrelic.com)
-* [Pingdom](http://pingdom.com)
+For synthetic transactions we recommend using setups that are close to your production tenant configuration as possible, potentially even using the same production tenant. Since setting up synthetic transactions with redirect flows and third party providers can be tricky, using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant) is recommended. This flow neither involves browser redirects nor requires a UI.
```

Comment:
For synthetic transactions, use an environment as close to your production tenant configuration as possible. You may potentially even use the same production tenant. Since setting up synthetic transactions with redirect flows and third party providers can be tricky, using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant) is recommended. This flow doesn't involve browser redirects or require a UI.

Reply:
will take some of this. I want to avoid using "environment" as we will be using that differently in the future